deepcell_label_utils/utils.py

```diff
@@ -93,14 +99,100 @@ def int_coords(x):
     return img_mask
 
 
+class DCLZipConverter:
+    """Label converter class for DeepCell Label zip to usable formats."""
+
+    def __init__(self, zip_path=None, zip_obj=None):
+        """
+        Load in a file path for a zip exported from DeepCell Label and
+        automatically reshape to BTZYXC.
+        """
+        if zip_path is not None:
+            zf = zipfile.ZipFile(zip_path, 'r')
+        elif zip_obj is not None:
+            zf = zip_obj
+        else:
+            print("No zip specified...")
+            return
+        data = zf.read('X.ome.tiff')
+        bytes_io = io.BytesIO(data)
+        X_init = TiffFile(bytes_io).asarray(squeeze=False)
+        self.X_ome = self.reshape_DCL(X_init)  # Reshape from _CTYX_ to BTZYXC
+        self.X = self.to_TYXC(self.X_ome)      # Reshape to TYXC for SLC
+
+        data = zf.read('y.ome.tiff')
+        bytes_io = io.BytesIO(data)
+        y_init = TiffFile(bytes_io).asarray(squeeze=False)
+        self.y_ome = self.reshape_DCL(y_init)  # Reshape from _CTYX_ to BTZYXC
+        self.y = self.to_TYXC(self.y_ome)      # Reshape to TYXC for SLC
+
+        data = zf.read('cells.json')
+        cells = json.loads(data.decode('utf-8'))
+        self.cells = cells
+        self.segments = self.segments_to_df(self.cells)  # To df for SLC
+
+        data = zf.read('divisions.json')
+        divisions = json.loads(data.decode('utf-8'))
+        self.divisions = divisions
```

Comment:
Is the division file always present even if there isn't a time dimension in the data?

Reply:
Yes, it'll just have an empty list.